packages/main/src/RadioButton.ts

```diff
@@ -102,8 +102,8 @@ class RadioButton extends UI5Element implements IFormInputElement {
 	/**
 	 * Defines whether the component is read-only.
 	 *
-	 * **Note:** A read-only component is not editable,
-	 * but still provides visual feedback upon user interaction.
+	 * **Note:** A read-only component is not editable (selectable),
+	 * but still provides visual feedback upon user interaction (is focusable).
```

Comment:
I suggest:
A read-only component isn't editable or selectable. However, because it's focusable, it still provides visual feedback upon user interaction.